mix.exs

```diff
@@ -55,7 +55,8 @@ defmodule Membrane.Live.Mixfile do
       {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
       {:credo, ">= 0.0.0", only: [:dev, :test], runtime: false},
       {:membrane_rtc_engine,
-       github: "membraneframework/membrane_rtc_engine", branch: "synchronize-streams-wallclock"},
+       github: "membraneframework/membrane_rtc_engine",
+       branch: "add-compositor-with-synchronization"},
```

Comment:
Are these dependency changes intentional?

Reply:
Added by mistake. I've changed it